bigtable/docs/snippets.py

```diff
@@ -160,6 +174,205 @@ def test_bigtable_create_app_profile():
+    column_family = table.column_family(column_family_id, gc_rule=gc_rule)
+    column_family.create()
+    # [END bigtable_create_column_family]
+
+    col_fams = table.list_column_families()
+    assert column_family_id in col_fams
+
+
+def test_bigtable_mutations_batcher_read_rows():
+    # [START bigtable_mutate_rows]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+    row_keys = [b'row_key_1', b'row_key_2', b'row_key_3', b'row_key_4']
+    col_name = b'col-name1'
+    cell_val = b'cell-val-abc'
+    # [END bigtable_mutate_rows]
+
+    expected_rows_data = []
+    for row in row_keys:
+        expected_rows_data.append(row.decode('utf-8'))
+        expected_rows_data.append(cell_val.decode('utf-8'))
+
+    # [START bigtable_mutate_rows]
+    rows = []
+    for row_key in row_keys:
+        row = table.row(row_key)
+        row.set_cell(COLUMN_FAMILY_ID, col_name, cell_val)
+        rows.append(row)
+
+    batcher = table.mutations_batcher(flush_count=3)
+
+    # In batcher mutate will flush current batch if it
+    # reaches the max flush_count
+    batcher.mutate_rows(rows)
+    batcher.flush()
+    # [END bigtable_mutate_rows]
+
+    # [START bigtable_read_rows]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    rows_data = table.read_rows()
+    # [END bigtable_read_rows]
+
+    actual_rows_data = []
+    for row in rows_data:
+        actual_rows_data.append(row.row_key.decode('utf-8'))
+        actual_rows_data.append(
+            row.cells[COLUMN_FAMILY_ID][col_name][0].value.decode('utf-8')
+            )
+
+    assert actual_rows_data == expected_rows_data
+
+    table.truncate(timeout=200)
+
+
+def test_bigtable_drop_by_prefix():
+    # [START bigtable_drop_by_prefix]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+    # [END bigtable_drop_by_prefix]
+
+    row_keys = [b'row_key_1', b'row_key_2', b'r_key_3', b'r_key_4']
+    rows = []
+    for row_key in row_keys:
+        row = table.row(row_key)
+        row.set_cell(COLUMN_FAMILY_ID, b'col-name1', b'cell-val-abc')
+        rows.append(row)
+    batcher = table.mutations_batcher()
+    batcher.mutate_rows(rows)
+    batcher.flush()
+
+    # [START bigtable_drop_by_prefix]
+    row_key_prefix = b'r_key'
+    table.drop_by_prefix(row_key_prefix, timeout=300)
+    # [END bigtable_drop_by_prefix]
+
+    rows_data = table.read_rows()
+    actual_rows_keys = []
+    for row in rows_data:
+        actual_rows_keys.append(row.row_key)
+
+    assert actual_rows_keys == [b'row_key_1', b'row_key_2']
+
+    table.truncate(timeout=300)
+
+
+def test_bigtable_mutate_rows_read_row():
+    # [START bigtable_mutations_batcher]
```

Comment:
You are not using `mutations_batcher` in this example

Reply:
Oh ok, got interchanged with mutate_rows.